src/cli/facade/clean.js

```diff
@@ -3,7 +3,6 @@
 const read = require('read');
 
 const strings = require('../../resources/index');
-const wrapCliCallback = require('../wrap-cli-callback');
```

Comment:
by doing this aren't we breaking contract for things like this:
https://github.com/opencomponents/grunt-oc/blob/master/tasks/oc-mock.js

Reply:
how? in that file it's explicitly passing a callback.

Reply:
we've chatted about this, the difference is about programmatic api, however we think that the use-case of not passing a callback in programmatic api does not make any sense. @matteofigus we're thinking of treating this as a non breaking change - is that ok?

Reply:
yeah makes sense, thanks